.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+name: Build release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: golang:1.19.4
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          [linux/amd64, linux/arm64, linux/386, darwin/amd64, darwin/arm64]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Compile
+        id: compile
+        run: |
+          echo ${{ matrix.platform }} | {
+            IFS='/' read -r GOOS GOARCH
+
+            export GOOS GOARCH CGO_ENABLED=0
```

Comment:
It switches off cross compiling for all apps.

Reply:
Yes, so that the build is reproducible not only for the airgapped app